projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-menu/wpcom-admin-menu.php

```diff
@@ -191,7 +191,7 @@ function wpcom_hide_customizer_submenu_on_block_theme() {
 
 		remove_action( 'customize_register', array( 'Jetpack_Fonts_Typekit', 'maybe_override_for_advanced_mode' ), 20 );
 
-		remove_action( 'customize_register', 'Automattic\Jetpack\Dashboard_Customizations\register_css_nudge_control' );
+		remove_action( 'customize_register', 'Automattic\Jetpack\Masterbar\register_css_nudge_control' );
```

Comment:
@fushar While fixing this I realized that https://github.com/Automattic/jetpack/pull/38233 essentially introduced functionality already present in `jetpack-mu-wpcom` via the [masterbar package](https://github.com/Automattic/jetpack/blob/21011d7ba80f243d02175286b33bbcec10e3f913/projects/packages/masterbar/src/admin-menu/load.php). Could you pls take a look and confirm if we actually need this as part of the `wpcom-admin-menu` feature?

Reply:
The PR that first introduced the duplicated logic is https://github.com/Automattic/jetpack/pull/36161. I believe, we ported the logic to jetpack-mu-wpcom because we're not using the masterbar class on Classic sites: https://github.com/Automattic/jetpack/blob/21011d7ba80f243d02175286b33bbcec10e3f913/projects/packages/masterbar/src/class-main.php#L32-L34

So yes, it's still needed here.

Reply:
Thanks for confirming! Ideally we could further cleanup/refactor our codebase so we don't introduce the same methods/logic in different packages. That said, I would like to confirm nothing breaks with my "fix" above. Could you please advise on how to manually test this change and for which environments (Simple, WoA, both)?

Reply:
hmm I just noticed though that we do load this file on Simple sites via the `masterbar` mu-plugin: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sznfgreone%2Snqzva%2Qzrah%2Sybnq.cuc%3Se%3Qonnnn97r%235-og

Reply:
Also fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sznfgreone.cuc%3Se%3Q19346oqp%2363-og

Reply:
I would say follow the test instructions in the original PR (https://github.com/Automattic/jetpack/pull/36161)

(The first two steps in the instructions are a bit obsolete; they just mean enable the Classic view from Settings -> General.)

Reply:
> hmm I just noticed though that we do load this file on Simple sites via the masterbar mu-plugin

Hmm, I just realized that. @mmtr, what do you think? 🤔

Reply:
> hmm I just noticed though that we do load this file on Simple sites via the masterbar mu-plugin

I took a look and it seems that it's needed to load the sidebar admin menu for sites with Default admin interface style.

Hmm... not sure. Maybe we should move that logic (loading the admin menu class) to jetpack-mu-wpcom instead, so that it can reference the class in the masterbar package? 🤔

Reply:
IIRC, when we first introduced Nav Unification, the `masterbar` logic was out of sync between Simple sites and Atomic sites.

Simple sites were (are?) using a mu-plugin, and Atomic sites were (are?) using the module from Jetpack. So we had load the admin-menu logic in both places.

Reply:
Nowadays, we have a single and common source of truth for Simple and Atomic sites: `jetpack-mu-wpcom`. So a good future step would be to consolidate everything there.

In the meantime, I don't think this fix breaks anything. I tested the sidebar menu in different scenarios (Simple Default, Simple Classic, Atomic Default, Atomic Classic, Self-hosted Jetpack) and in different environments (Calypso, WP Admin) and couldn't spot any regression.

Reply:
Yeah, Simple sites load the file from the Jetpack masterbar package, which is not being changed in this PR.